src/ui/coordinates/coordView.ts

```diff
@@ -0,0 +1,103 @@
+import h from 'mithril/hyperscript'
+import Chessground from '~/chessground/Chessground'
+import redraw from '~/utils/redraw'
+import { header } from '../shared/common'
+import Board from '../shared/Board'
+import i18n from '../../i18n'
+import layout from '../layout'
+import { INITIAL_FEN } from 'chessops/fen'
+import CoordCtrl from './coordCtrl'
+
+let orientation = (Math.random() > 0.5 ? 'white' : 'black') as Color
+let random = true
+
+function reOriente(color: Color | undefined): void {
```

Comment:
that function could be in the controller I think

Reply:
It modifies the view property `orientation`  and then redraw, I don't think it can be moved without a lot of complications. But more importantly I don't think it belongs to the controller.

Reply:
I moved the code to chessground and used similar functions